src/components/organisms/TemplateModal/TemplateModal.tsx

```diff
@@ -51,6 +53,7 @@ const TemplateModal: React.FC<TemplateModalProps> = props => {
   const onClickSubmit = useCallback(
     (formDataList: Record<string, Primitive>[]) => {
       if (projectToCreate) {
+        trackEvent('START_FROM_A_TEMPLATE', {templateID: template.id});
```

Comment:
should we track exactly which template was used or just the event name?

Reply:
@devcatalin both. In that way we can remove the least needed.
Actions are defined here: https://kubeshop.atlassian.net/wiki/spaces/KUBESHOP/pages/436043777/Telemetry#Actions